backend/dev-config.yaml

```diff
@@ -16,5 +16,5 @@ db:
     mode: "enable"
     init-script: "./sql/init.sql"
 
-# Other possible values: 'Prod' and 'Nothing'.
-logging: Dev
+# Other possible values: "prod" and "nothing".
+logging: "dev"
```

Comment:
Oh, while we are at it, there is an old comment a few lines above:
>   # It should match parameters from `/deployment/docker-compose.yml`

It's no longer the case, now these options don't have to match `/deployment/docker-compose.yml` because we have a completely separate config for `deployment`. Can you please fix it (it's unrelated to your issue, but it's a trivial change, so I think it's ok to do it here)?

Reply:
sure